packages/glimmer-runtime/tests/updating-test.ts

```diff
@@ -1793,3 +1793,68 @@ test("HTML namespace is continued to child templates", function() {
   equal(getSvg().namespaceURI, SVG_NAMESPACE);
   equal(getSvg().firstChild.namespaceURI, SVG_NAMESPACE);
 });
+
+QUnit.module("Updating Element Modifiers", {
+  setup: commonSetup
+});
+
+test("Updating a element modifier", assert => {
+  let manager = new TestModifierManager();
+  env.registerInternalModifier('foo', manager);
```

Comment:
Don't need to call it "internal". The diff in helper is internal helpers work with references and the user helpers are just functions. It's not a glimmer distinction. Just how the TestEnvironment is set up. For modifiers we don't have that difference